ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -102,8 +102,21 @@ gcp_connection::delete_object(const std::string &object_key)
 
 // Retrieves an object from the google cloud bucket.
 int
-gcp_connection::get_object(const std::string &object_key, const std::string &path) const
+gcp_connection::read_object(const std::string &object_key, int64_t offset, size_t len, void *buf)
 {
+    namespace gcs = ::google::cloud::storage;
```

Comment:
this has already been declared at the top of the file so this line is redundant

Reply:
fixed: 51bdb8037abbb5200d843c2dcb7ae667734c3448